docs/dev-notes.md

```diff
@@ -26,6 +27,28 @@ should get you a shell inside a docker container with all of
 spamgourmet set up and with all of the spamgourmet code visible to
 allow you to test code you are working on.
 
+Git and Git Branching
+#####################
+
+We use Git as our source code repository management system. This is quite
+powerful and sophisticated and so sometimes a little intimidating to start
+with. Despite that, what you need to contribute to spamgourmet will be quite
+simple. If you don't know git then this is likely a good place to start
+
+https://docs.github.com/en/get-started/using-git
+
+For any changes you make,
```

Comment:
is some text missing here?

Reply:
Remove that line unless you want to expand on it?